src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,34 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        StringBuilder informationAboutSalary = new StringBuilder("Отчёт за период ")
+                .append(dateFrom).append(" - ").append(dateTo);
+        LocalDate startData = LocalDate.parse(dateFrom, formatter);
+        LocalDate finishData = LocalDate.parse(dateTo, formatter);
+        if (startData.compareTo(finishData) > 0) {
```

Comment:
LocalDate class has methods: isAfter and isBefore

Reply:
Already read it, thanks. Done.